charts/brigade/values.yaml

```diff
@@ -35,7 +35,7 @@ apiserver:
   thirdPartyAuth:
     ## Valid values are "oidc" (for OpenID Connect), "github" (for OAuth2 with
     ## GitHub as the identity provider), or "disabled"
-    strategy: disabled
+    strategy: github
```

Comment:
I don't think we want these changes to go in, do we 😄 ?

I'd suggest leaving with `disabled` as the default still, but definitely keeping the new `admins` section/comments, perhaps with a sample commented-out GH username.

Reply:
Yep. That was an accident! Thanks for catching!

Reply:
Fixed after force push.